server/app/tool/readme.md

```diff
@@ -0,0 +1,119 @@
+# Tool
+
+Currently, `tool` is a binary package written in Rust. It mainly provides the following functions:
+- Set/Get each TEE's Open Policy Engine(OPA) `Policy(.rego)` and `Reference Data(.json)` files. The `Reference Data(.json)` contents should come from such as Reference Value Provider Service(RVPS), but this item is not reflected in this implementation.
+- Restore each TEE's `Policy(.rego)` and `Reference Data(.json)` to default value.
+- Provide the Attestation Server's `attestation` endpoint testing functionality.
+
+## Supported TEEs
+
+Currently the `tool` supports the following types of TEE:
+- sgx
+- tdx
+- sample: The TEE used to demo/test Attestation Server's functionalities.
+
+## Commands
+
+The `tool` supports each TEE's `Policy(.rego)` and `Reference Data(.json)` configuration and testing subcommands, as the following:
+- get-policy
+- set-policy
+- get-reference-data
+- set-reference-data
+- restore-default-policy
+- restore-default-reference-data
+- test-attestation
+
+### get-policy
+
+Get the <TEE> specific evaluation `Policy(.regp)` from Attestation Server."
```

Comment:
1. note that the `<TEE>` inside regular markdown text is not rendered (it is consider as an raw-HTML, and `<TEE>` is not a valid HTML tag). So either wrap it with backticks, or remove the angle-bracket, like:

    Get the TEE-specific evaluation `Policy(.rego)` from the Attestsion Server.

2. typo: .regp -> .rego
3. remove final double quotes

(these comments apply to several commands below)